backend/src/services/unified_guide_service.py

```
@@ -0,0 +1,1287 @@
+                "cultural_insights": {},
+                "transportation": {},
+                "safety_info": {},
+                "parsing_error": str(e)
+            }
+
+    async def _fetch_google_places_restaurants(self, context: GuideGenerationContext) -> List[Dict[str, Any]]:
+        """Fetch restaurants using Google Places API with persona-based filtering"""
+        try:
+            await self.google_places_service.initialize()
+            
+            persona_config = self.prompts["travel_guide"]["persona_adaptation"][context.persona]
```

Ruff (0.13.1)

Local variable `persona_config` is assigned to but never used

Remove assignment to unused variable `persona_config`

(F841)